exercises/book-store/example.rs

```diff
@@ -96,10 +93,9 @@ fn hash_of(basket: &GroupedBasket) -> u64 {
 
 pub fn lowest_price(books: &[Book]) -> Price {
     DecomposeGroups::new(books)
-        .map(|gb| r64(basket_price(&gb)))
```

Comment:
immediate reaction: Huh? Where did `r64` come from?

After reading docs: Oh, it's https://docs.rs/noisy_float/0.1.4/noisy_float/types/fn.r64.html . Carry on then.

Reply:
TBH it caught me the same way just now when I was looking over the code. It's useful that this PR eliminates that dependency anyway, because if it hadn't, it may have been worth filing a follow-up issue to clarify what was happening there.